content/shmem_reductions.tex

```diff
@@ -125,8 +125,9 @@
     \dest{} array on all \acp{PE} in the active set.  The active set is defined
     by the \VAR{PE\_start}, \VAR{logPE\_stride}, \VAR{PE\_size} triplet.
 
-    The \source{} and \dest{} arrays may be the same array, but they may not be
-    overlapping arrays.
+    The \source{} and \dest{} arrays must not point to partially overlapping
```

Comment:
I feel like "region" isn't a precisely defined term, and it isn't very commonly used in the spec (looks like in 1.4 it's only used in shmem_calloc and in the locking APIs).

After reading the changes you've made to the symmetric pointer text, I'm wondering if it would be better to have this change build on that and say something like:

The \source{} and \dest{} arguments must either be passed the same symmetric address, or two different symmetric addresses at least \nreduce{} elements apart. That is, they must be completely overlapping or completely disjoint.

Reply:
I like this change.  The `nreduce` bit doesn't completely address the case where a pointer queried via `shmem_ptr` aliases the other pointer.  We could borrow the overlapping buffer text from #276.  What do you think?
> The \source{} and \dest{} arguments must either be the same symmetric address, or two different symmetric addresses corresponding to buffers that do not overlap in memory. That is, they must be completely overlapping or completely disjoint.